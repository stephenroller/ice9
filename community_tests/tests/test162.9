# 2 procs w/ same name									fail

proc foo(a,b : int) : int
end

proc 
foo
(a,b : int) : int
end
