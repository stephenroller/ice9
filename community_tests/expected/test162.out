line 7: proc foo has already defined in the current scope